### PR TITLE
Added used and available storage to SettingsFragment

### LINKED ISSUE #46: Show actual used space as given by the block server

### PR DESCRIPTION
**Pull request by L-Henke**
Since there was no mockup available, used and available storage has been added to SettingsFragment.

If returned quota is "0" (as it currently is) "unlimited storage" is shown as available storage ("Used: 5MB of unlimited storage"). @jmt85 do you want an empty string instead ("Used: 5MB")?

Closes #46

### CODE REVIEW

File: qabelbox/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -191,4 +191,9 @@
     <string name="message_cant_attach_external_file_exists">Anfügen der freigegebenen Datei nicht möglich. Eine Datei mit diesem Namen existiert bereits. Vielleicht haben Sie diese Datei schon ein Mal früher angefügt</string>
     <string name="hockeyapp_feedback_response_button_text">Neue Nachricht senden</string>
     <string name="action_about">Über</string>
+   <string name="used_storage">Belegt: %1$s von %2$s</string>
+   <string name="unlimited_storage">unbegrentem Speicherplatz</string>
```
Comment (audax) [inline qabelbox/src/main/res/values-de/strings.xml]: *unbegrenzt

Comment (L-Henke) [inline qabelbox/src/main/res/values-de/strings.xml]: Actually I am pretty sure that the case is grammatically correct. Even the extraordinary good spell check "duden.de" states the same "Belegt: 5 MB von unbegrenztem Speicherplatz." while "Belegt: 5 MB von unbegrenzt Speicherplatz" generates the error: "Überprüfen Sie, ob hier wegen der vorangehenden Präposition ein anderer Kasus (Fall) verlangt wird."

But maybe you just wanted to state that a "z" is missing. 

The more relevant question is, do we want to translate a quota of "0" as unlimited storage.

**Comment by L-Henke** [comment]
Since the updated diff hides my comment, I'm going to quote myself:

> The more relevant question is, do we want to translate a quota of "0" as unlimited storage.

**Comment by jmt85** [comment]
please merge master into

**Comment by jmt85** [comment]
why should be use 5 MB?

**Comment by L-Henke** [comment]
>translate 0 to "unlimited storage" is principle fine, but the message should read "Used: 5MB" without unlimited storage.

> Is this behaviour common? If this is the case we must talk to the marketing.

It seems like you _completely_ changed this comment, because my E-Mail notification contains a completely different text. I always strike through (~~example~~) if I modify a text to not completely confuse other people.

Your comment doesn't really help, because you are saying that the term unlimited storage is fine, but you don't want it AND you want to talk to the marketing about it which is very controversial.

The way more technical question is, why does the accounting server gives "0" as the quota although a concrete quota was defined month ago if I recall correctly. Maybe @audax knows this?

**Comment by audax** [comment]
The accounting server will not know the quota anyway. The whole REST resource was moved to the block server and is not yet implemented. The priority of this ticket should have been very low.

The plan was to only implement this in the desktop application and then integrate it into the android client in the next version since we are low on devs at the moment.

Please let this PR sit a while until I give it a go.

File: qabelbox/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -258,5 +258,11 @@
     <string name="github_url">https://github.com/Qabel</string>
     <string name="asset_qabel_font">fonts/Art Post.ttf</string>
     <string name="help">Help</string>
+    <string name="settings_box_quota" translatable="false">show_box_quota</string>
```
Comment (dannywerner) [inline qabelbox/src/main/res/values/strings.xml]: pls rename to key_settings_box_quota

**Comment by audax** [comment]
We will dig this code out later, if needed.

**Pull request closed without merging**